### ISSUE TITLE
Expanded model rows in profile not particularly useful.

### ISSUE THREAD

**Issue by kadams54** [juju/juju-gui#2506]
At the moment the expanded model rows don't display any additional information; the only reason to expand one is to get to the manage and delete buttons. Maybe time to revisit the original design and see if we can add more of the original vision (lists of errors, recent changes, an SVG rendering of the model, similar to what we do with bundles, etc.) in? Right now it's kinda painful to have to click twice just to get to the manage button.

**Comment by frankban** [comment]
+1, maybe we can simplify the component

**Comment by spencerbygraves** [comment]
Sounds good. When you're ready, we can check the design to make sure the styles are up to date.

**Comment by kadams54** [comment]
I've been mucking around in this code working on other bugs (#2338) and I think there is prerequisite work that needs to be done before this issue. Right now, some of the lists in the profile page (model, charm, and bundle) display each row as a UserProfileEntity component. There are several problems with this approach: 1) The model is not, as normally understood within the GUI, an entity. That term is typically used for just charms and bundles. 2) We're somewhat constrained to generic logic and display that applies across all three, very different object types, which makes customizing the content difficult.

In order to add a slew of model-specific content (for this issue), we need to first split model-related code out of UserProfileEntity. We may also want to make further architectural changes, such as having the new, model-specific component inherit from a base component that is shared with the charm and bundle. There's probably enough work there to merit its own issue, though I'd like to hear what others (@hatched ) think before creating the issue.

**Comment by hatched** [comment]
IIRC we decided to go with the shared component for everyone for rapid development. Now that we're shaking out the feature set of the profile, I agree it makes sense to split them out into more appropriate components. Before making these changes though we should revisit the designs and correct any invalid assumptions done prior to the implementation.

**Comment by hatched** [comment]
Closing this as we're implementing a new UI
